lists-ui/src/locale/en.json

```diff
@@ -1,14 +1,14 @@
 {
-  "CONSERVATION_LIST": "Conservation List",
-  "SENSITIVE_LIST": "Sensitive List",
-  "COMMON_TRAIT": "Traits List",
-  "LOCAL_LIST": "Local List",
-  "SPECIES_CHARACTERS": "Character List",
-  "INVASIVE": "Invasive species List",
-  "COMMON_HABITAT": "Common Habitat List",
-  "PROFILE": "Profile List",
-  "TEST": "Testing List",
-  "OTHER": "Other List",
+  "CONSERVATION_LIST": "Conservation list",
+  "SENSITIVE_LIST": "Sensitive list",
+  "COMMON_TRAIT": "Common traits list",
```

Comment:
Changed sort to use facet type name, so it is consistent but order was wrong when displaying these i18n codes, so made them consistent with the enum values. Can change the values to match expected human labels if needed.